addons/medical_engine/functions/fnc_handleDamage.sqf

```diff
@@ -83,11 +87,8 @@ if (
 
 // Receiving explosive damage inside a vehicle doesn't trigger for each hitpoint
 // This is the case for mines, explosives, artillery, and catasthrophic vehicle explosions
-// Triggers twice, but that doesn't matter as damage is low
 if (
-    _hitPoint isEqualTo "#structural" &&
-    {!isNull _vehicle} &&
-    {_ammo isNotEqualTo ""} &&
+    (_structuralDamage && !_environmentDamage && _inVehicle) &&
```

Comment:
Just to match the order in line 78.
```suggestion
    (!_environmentDamage && _inVehicle && _structuralDamage) &&
```

Reply:
I guess it doesn't matter since this set isn't lazy-eval'd anymore, but the order was intentional for fast failure